gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/RateLimiterConfiguration.java

```diff
@@ -39,6 +39,17 @@
    */
   private int rateSeconds = 10;
 
+  /**
+   * The registration order for {@link com.netflix.spinnaker.gate.ratelimit.RateLimitingFilter}.
+   *
+   * <p>100 - Run after the spring security filter chain. -100 - Run prior to the spring security
+   * filter chain.
+   *
+   * <p>When the filter is run prior to the spring security filter chain, it will only rate limit
```

Comment:
We talked about this potentially being sufficient, thoughts?

If it become problematic to run early/ignore sso requests, we could also wrap another filter that ran post security chain.

Reply:
Yeah, this seems absolutely the right call.